### ISSUE TITLE
Not booting. Flashing cursor installation

### ISSUE THREAD

**Issue by ronbarbosa** [blend-os/blendOS#62]
Using the KDE iso. The installation seems to complete. I can confirm about 2GB of data on the target disk. There are errors during the installation but they flash quickly onscreen and do not display in the post-install log viewer.

When the installer reboots, the system shows a text-based screen with a flashing cursor in the upper left corner and hangs there. No error text is shown and I cannot switch to an alternate terminal to attempt to log in and diagnose the problem.

The installer will only boot in legacy bios mode, which seems to force the installation itself to be in legacy bios mode, which cannot be changed during the installation.

Looks like a great OS. But I can't seem to test it out myself.

**Comment by theblackapple5150** [comment]
Same issue here with gnome installer

**Comment by DET171** [comment]
Same here

**Comment by ronbarbosa** [comment]
> 

Hey all, I was able to get this to work eventually. It seems that this build just will not boot in Legacy BIOS mode at all. I was able to force it to boot in UEFI by disabling legacy BIOS fallback in my motherboard BIOS.

That gets the installation working in UEFI mode, BUT the installation still fails to create a proper UEFI application to actually allow the motherboard BIOS to recognize it as a viable UEFI installation.

I was able to complete the installation and then reboot BACK into the installer's live ISO, and through trial and error, I was able to cobble together a proper UEFI setup. The installer does create a UEFI partition and puts some files/folders in there but it doesn't put enough data in there.

For starters, let the installer manage the partitioning. Just choose the destination disk you'd like. Reboot the installer ISO which will give you root and allow you to prep the destination disk to actually function. Then make the partition table look like this, by changing the partition type for the (smaller) boot partition to EFI (option 1 in gparted) and adding a label of "blendOS" to the larger root file system partition (make sure you add the label):

```
Device           Start        End    Sectors   Size Type
/dev/nvme0n1p1      34     976562     976529 476.8M EFI System
/dev/nvme0n1p2 1368064 1953523711 1952155648 930.9G Linux filesystem
```

From there you want to mount the ISO's 2nd partition on a temporary mount point and the destination disk's UEFI partition on a separate mount point and start copying files from the ISO partition to your destination disk's UEFI partition until the destination's UEFI partition looks like this (you're copying only ... do not delete anything that was already on the destination's UEFI partition):

```
drwxr-xr-x root/root         0 2023-04-30 22:28 ./efi/
drwxr-xr-x root/root         0 2023-04-30 22:27 ./efi/BOOT/
-rwxr-xr-x root/root     92160 2023-04-30 22:27 ./efi/BOOT/BOOTx64.EFI
-rwxr-xr-x root/root   5623808 2023-04-30 22:27 ./efi/BOOT/BOOTIA32.EFI
drwxr-xr-x root/root         0 2023-04-30 21:55 ./grub/
-rwxr-xr-x root/root      4638 2023-04-30 21:55 ./grub/grub.cfg
-rwxr-xr-x root/root  16732380 2023-04-30 21:54 ./initramfs-linux.img
-rwxr-xr-x root/root  12507968 2023-04-30 21:54 ./vmlinuz-linux
drwxr-xr-x root/root         0 2023-04-30 23:15 ./loader/
-rwxr-xr-x root/root       117 2023-04-30 22:38 ./loader/loader.conf
drwxr-xr-x root/root         0 2023-04-30 23:42 ./loader/entries/
-rwxr-xr-x root/root       169 2023-04-30 23:42 ./loader/entries/blendos-x86_64-linux.conf
-rwxr-xr-x root/root        54 2023-04-30 23:15 ./loader/230501025318.BMP
-rwxr-xr-x root/root        54 2023-04-30 23:15 ./loader/230501025319.BMP
-rwxr-xr-x root/root     51200 2023-04-30 22:51 ./amd-ucode.img
-rwxr-xr-x root/root   7066624 2023-04-30 22:51 ./intel-ucode.img

```

Finally, edit loader/entries/blendos-x86_64-linux.conf to look like this (this is where we use the "blendOS" label):

```
title    blendOS (x86_64, UEFI)
linux    /vmlinuz-linux
initrd   /intel-ucode.img
initrd   /amd-ucode.img
initrd   /initramfs-linux.img
options  root="LABEL=blendOS" rw

```

Finally, reboot and your BIOS should recognize this installation is a proper UEFI bootable OS.

Good luck!

**Comment by RudraSwat** [comment]
This should be fixed in the latest ISO.

**Comment by ronbarbosa** [comment]
> This should be fixed in the latest ISO.

@RudraSwat can you please confirm whether you mean the issue is fixed in the latest AVAILABLE IOS or the latest UPCOMING ISO?

The issue I reported was against the ISO that is currently available to download: blendOS-2023.04.22-x86_64-plasma.iso

Do you know when we cab expect a new ISO that includes the fix?

**Comment by meika** [comment]
hello came here because I manage to get the installer to run after booting the image on the USB drive from non-uefi in boot menu, it booted ok... but I now have the flashing underscore cursor, 

this was because when booting from eufi or uefi, and booted fine and connected to the internet (ethernet and wireless once I logged it to it) 

but the installer failed to recognise that the internet was in fact working even as I could surf the web using firefox, 

I'll come back later to blend OS when this is not so... frictional... this is on my distro grasshopper intel NUC DN282OFYKH with a celeron inside....
